src/index.js

```diff
@@ -0,0 +1,45 @@
+import './style.css';
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
+
+const toDo = [
+  {
+    description: 'clean my room',
+    comleted: false,
+    index: 1,
+  },
+  {
+    description: 'check my emails',
+    comleted: true,
+    index: 2,
+  },
+  {
+    description: 'read my book',
+    comleted: false,
+    index: 3,
+  },
+  {
+    description: 'working on my coding skills',
+    comleted: false,
```

Comment:
- [ ] You made some typo errors here. It should be `completed`, not `comleted`. Kindly fix that

Reply:
changed